### DISCUSSION TITLE
How to interact with extension from JS ?

### DISCUSSION THREAD

**Discussion by Hateman31** [aclap-dev/video-downloadhelper#1448] [Help and Q&A]
### Your question

I want to using VDapp under the Selenium Web Driver. Is it possible to send actions to extension through JS ?

### Addon & CoApp version and details

Version 9.1.0.44 (stable), target mozilla, locale en-US.
Companion App installed (v2.0.19)

**Comment by jcfrog** [comment]
I'm afraid not.
